### PR TITLE
Fix miss text

### PR DESCRIPTION
**Pull request by xiliu8006**
Fix a bug, Miss text file when generating test file which will be submitted to test server! #123

### CODE REVIEW

File: tests/test_dataset/test_kitti_dataset.py [hunk 1]
```diff
@@ -306,6 +308,98 @@ def test_format_results():
     assert np.allclose(result_files[0]['sample_idx'], expected_sample_idx)
 
 
+def test_bbox2result_kitti():
+    data_root = 'tests/data/kitti'
+    ann_file = 'tests/data/kitti/kitti_infos_train.pkl'
+    classes = ['Pedestrian', 'Cyclist', 'Car']
+    pts_prefix = 'velodyne_reduced'
+    pipeline = [{
+        'type': 'LoadPointsFromFile',
```
Comment (ZwwWayne) [inline tests/test_dataset/test_kitti_dataset.py:317]: Can we use dict() to make this part simpler and following the convention?

File: tests/test_dataset/test_kitti_dataset.py [hunk 2]
```diff
@@ -306,6 +308,98 @@ def test_format_results():
...
+    modality = {'use_lidar': True, 'use_camera': False}
+    split = 'training'
+    self = KittiDataset(
+        data_root,
+        ann_file,
+        split,
+        pts_prefix,
+        pipeline,
+        classes,
+        modality,
```
Comment (ZwwWayne) [inline tests/test_dataset/test_kitti_dataset.py:360]: clean comma.

File: tests/test_dataset/test_kitti_dataset.py [hunk 3]
```diff
@@ -306,6 +308,98 @@ def test_format_results():
...
+        torch.tensor(
+            [[8.7314, -1.8559, -1.5997, 0.4800, 1.2000, 1.8900, 0.0100]]))
+    labels_3d = torch.tensor([
+        0,
+    ])
+    scores_3d = torch.tensor([0.5])
+    result = dict(boxes_3d=boxes_3d, labels_3d=labels_3d, scores_3d=scores_3d)
+    results = [result]
+    temp_kitti_result_dir = tempfile.mkdtemp()
+    print(temp_kitti_result_dir)
```
Comment (ZwwWayne) [inline tests/test_dataset/test_kitti_dataset.py:372]: unnecessary.

File: tests/test_dataset/test_kitti_dataset.py [hunk 4]
```diff
@@ -306,6 +308,98 @@ def test_format_results():
...
+    self = KittiDataset(
+        data_root,
+        ann_file,
+        split,
+        pts_prefix,
+        pipeline,
+        classes,
+        modality,
+    )
+    boxes_3d = LiDARInstance3DBoxes(
```
Comment (ZwwWayne) [inline tests/test_dataset/test_kitti_dataset.py:362]: May also test empty boxes to verify the bug is fixed.

Comment (xiliu8006) [reply tests/test_dataset/test_kitti_dataset.py:362]: The empty boxes test code at line [387](https://github.com/xiliu8006/mmdetection3d/blob/fix_miss_text/tests/test_dataset/test_kitti_dataset.py#L388)

File: tests/test_dataset/test_kitti_dataset.py (hunk 3 above)
Comment (xiliu8006) [reply tests/test_dataset/test_kitti_dataset.py:372]: I am sorry to forget to delete the debug code

File: tests/test_dataset/test_kitti_dataset.py [hunk 5]
```diff
@@ -228,52 +204,39 @@ def test_format_results():
...
-            'type': 'DefaultFormatBundle3D',
-            'class_names': ['Pedestrian', 'Cyclist', 'Car'],
-            'with_label': False
-        }, {
-            'type': 'Collect3D',
-            'keys': ['points']
-        }]
-    }]
-    modality = {'use_lidar': True, 'use_camera': False}
+    pipeline = [
```
Comment (ZwwWayne) [inline tests/test_dataset/test_kitti_dataset.py:207]: It seems that similar pipeline are generated and used multiple times. Can we implement a new function to generate it and reduce code?

Comment (xiliu8006) [reply tests/test_dataset/test_kitti_dataset.py:207]: For sure

File: tests/test_dataset/test_kitti_dataset.py (hunk 3 above)
Comment (ZwwWayne) [reply tests/test_dataset/test_kitti_dataset.py:222]: ping

**Review by ZwwWayne — APPROVED** [review]
